stdlib/3/urllib/request.pyi

```diff
@@ -140,9 +140,10 @@ class ProxyBasicAuthHandler(AbstractBasicAuthHandler, BaseHandler):
 
 class AbstractDigestAuthHandler:
     def __init__(self,
-                 password_mgr: Optional[HTTPPasswordMgr] = ...) -> None: ...
+                 passwd: Optional[HTTPPasswordMgr] = ...) -> None: ...
     def http_error_auth_reqed(self, auth_header: str, host: str, req: Request,
                               headers: Mapping[str, str]) -> None: ...
+    def get_authorization(self, req, chal): ...
```

Comment:
Yep. Same as in Python 2.  https://github.com/python/typeshed/blob/092b15e3e9ff376b7c141e137fcbd3397f08074a/stdlib/2/urllib2.pyi#L107

I can try to give it types but don't know how correct they'll be.